frontend/src/components/LoginAndRegistrationPage/LoginPage/LoginForm.js

```diff
@@ -24,21 +25,8 @@ export default function LoginForm() {
                 <Form>
                     <Container>
                         <Row className="mx-auto">
-                            {/* //TODO: pomyśleć o funkcji generującej Col */}
-                            <Col className="form-group" xs={12}>
-                                <h6>Email</h6>
-                                <Field className="form-control" type="email" name="email" />
-                                <ErrorMessage name="email" component="div">
-                                    {msg => <div style={{ color: 'var(--font-color)' }}>{msg}</div>}
-                                </ErrorMessage>
-                            </Col>
-                            <Col className="form-group" xs={12}>
-                                <h6>Hasło</h6>
-                                <Field className="form-control" type="password" name="password" />
-                                <ErrorMessage name="password" component="div">
-                                    {msg => <div style={{ color: 'var(--font-color)' }}>{msg}</div>}
-                                </ErrorMessage>
-                            </Col>
+                            {FormCol('Email', 'email', 'email')}
+                            {FormCol('Hasło', 'password', 'password')}
```

Comment:
Nie jestem przekonany czy to zadziała gdy zechcesz się zalogować. Tzn gdy już klikniesz "zaloguj" to wartości zebrane do `values` są przekazywane w `onSubmit` na backend. Gdy zrobisz jak powyżej to nie jestem pewien czy ten email i password znajdzie się w tym values, który zostanie wykorzystany do zalogowania. Możesz to łatwo sprawdzić: dodaj sobie onSubmit i zrób tam 
``` js
console.log(values)
```
I jeżeli po kliknięciu w zaloguj ten console.log wypisze te dane to ok, nie mam wtedy zastrzeżeń. Ale jeżeli nie zadziała to sugeruję powrót do poprzedniego rozwiązania

Reply:
console.log(values) nie wypisuje nic w submit, ale nie wypisuje też na developie. Widać te dane w adresie po submit, ale nie przekazuje dalej. Zaraz wkopię się w docsy formika

Reply:
Żeby onSubmit się odpalał, trzeba zrobić coś takiego: 
```javascript
{({ isSubmitting, values, errors, handleSubmit }) => (
                <Form onSubmit={handleSubmit}>
```

na początku formki - bez handleSubmit przekazanego z formika leci default event -> przekazanie przez paramsy i odświeżenie strony. Podmienię tak żeby normalnie pokazywały się alerty w Login i RegisterForm, powinno potem działać.

Reply:
Widzę teraz że walidacja jest źle, wkopię się w to jeszcze

Reply:
Nie działa, bo do listy errorów dodaje np email: "" przy poprawnej walidacji maila i nie przechodzi poprawny submit

Reply:
to mega dziwne w takim razie, to spróbuj naprawić jeżeli możesz i daj znać co było nie tak ;)

Reply:
spk, pracuję nad tym